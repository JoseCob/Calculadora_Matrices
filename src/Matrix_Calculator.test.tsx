import React from 'react';
import {render, screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import '@testing-library/jest-dom'; //Importamos la aserci√≥n personalizada
import matrixStructure from '../src/Matrix_Calculator'; //Importamos el componente para aplicarle el test

