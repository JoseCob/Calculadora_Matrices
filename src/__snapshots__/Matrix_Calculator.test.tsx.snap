// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MatrixStructure > coincide con la instantánea 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Estructura de la matriz
    </h1>
    <p>
      <strong>
        Nota*: 
      </strong>
       Solo se puede sacar la inversa de una matriz y su multiplicación cuando sea cuadrada,
    </p>
    <p>
      de lo contrario se desabilitara las opciones para realizar esas operaciones
    </p>
    <p />
    <div
      class="range-container"
    >
      <p>
        Matriz A
      </p>
      <label
        class="label-matrix"
      >
        Filas: 
      </label>
      <input
        class="input-matrix"
        type="text"
        value="2"
      />
      <label
        class="label-matrix"
      >
        Columnas: 
      </label>
      <input
        class="input-matrix"
        type="text"
        value="2"
      />
    </div>
    <div
      class="range-container"
    >
      <p>
        Matriz B
      </p>
      <label
        class="label-matrix"
      >
        Filas: 
      </label>
      <input
        class="input-matrix"
        type="text"
        value="2"
      />
      <label
        class="label-matrix"
      >
        Columnas: 
      </label>
      <input
        class="input-matrix"
        type="text"
        value="2"
      />
    </div>
    <button
      class="btn-createMX"
    >
      Crear Matriz
    </button>
  </div>
</DocumentFragment>
`;
